common/oatbox/user/UserLanguageService.php

```diff
@@ -64,7 +64,11 @@ public function getInterfaceLanguage(User $user)
         }
 
         $lang = $user->getPropertyValues(GenerisRdf::PROPERTY_USER_UILG);
-        return empty($lang) ? $this->getDefaultLanguage() : (string)current($lang);
+        if(!empty($lang)) {
+            return (string)current($lang);
+        }
+
+        return $this->getOption(self::OPTION_DEFAULT_LANGUAGE, DEFAULT_LANG);
```

Comment:
We generally add a blank line here. Can simplify return. Totally option, just cosmetics.
```suggestion

        return empty($lang) ? $this->getOption(self::OPTION_DEFAULT_LANGUAGE, DEFAULT_LANG) : (string)current($lang);
        }
```